azkaban-common/src/main/java/azkaban/executor/container/ContainerCleanupManager.java

```diff
@@ -438,10 +493,8 @@ public void start() {
         this.executionCleanupIntervalMin, TimeUnit.MINUTES);
     this.cleanupService.scheduleAtFixedRate(this::cleanUpContainersInTerminalStatuses, 0L,
         this.containerCleanupIntervalMin, TimeUnit.MINUTES);
-
-    // TODO: currently run-once for development; change to a fixed schedule after development
-    //  complete
-    this.cleanupService.schedule(this::cleanUpDanglingYarnApplications, 0L, TimeUnit.MINUTES);
+    this.cleanupService.scheduleAtFixedRate(this::cleanUpDanglingYarnApplications, 0L,
```

Comment:
Could you please write a unit test for cleanUpDanglingYarnApplications function?

Reply:
sure, will do